empirical_photometry.ipynb

```diff
@@ -0,0 +1,467 @@
atNZMmTaJChQrExMQQEBBAr169UlR/zvNIsSOEECJFSzgj8meffUamTJmMjvRGwsLCaNasGV9//TXVq1cnODiYcuXKGR3LJkjPjhBCiBQruTQiHzt2DB8fH86cOcNPP/3EN998g4ND8h7PiI6OtvjY5P0nIYQQQrxEv379Hjci22Oho7Vm5syZlC1blrCwMLZt28Z//vOfZF3oxMTE8PPPP/PJJ59Y/Jzk+6chhBBCvMCjR48AmDZtGvv377fLRuTw8HB8fX1p164dFSpUIDg4GG9vb6NjWdWZM2fw8vJi4MCBFC9e3OLnSbGTRC7cDue7VccpMcSfAv9ZR4kh/ny36jgXbocblun8+fN88cUXZMqUicyZM/P5559z7tw58ufPT+XKlQ3LJYQQ1hLfiFyrVi2ioqLImTMnhQsXNjrWazt16hRly5Zlzpw5DBkyBH9/f9555x2jY1ndw4cPOX/+PAsXLmTJkiUWP096dpLA9r9v0HXeYaJNZmLMGoAHj2JYdOASyw9dYVJLN6oUSdrpx2/fvo2XlxfXr1+nc+fOFC1alICAAKpUqUJ4uHEFmBBCWEvCGZF79Ohht5d65s+fT6dOnUiXLh3+/v7UqFHD6EhWdeHCBZYvX06fPn34+OOPOXfuHGnSpHmt17DPv2k7cuF2OF3nHSYi2vS40IkXY9ZERJvoOu9wko/wjBw5ksuXLzNr1izGjx9Ply5dWLBgAY0bN+bWrVtJmkUIIazt7NmzlC9fns2bNzNlyhTGjRuHk5N9fd6PjIykU6dOtGzZEjc3N44cOZKsC534fqSSJUsyZMiQx4uxvm6hA1LsWN30gBCiTeaXHhNtMvPfgHNJlCjWmjVrePfdd2nWrNkT2/v165ekOYQQwtq01jRp0oRr166xadMmOnXqZHSk13bmzBnKly/PtGnTGDBgANu2beO9994zOpbVXLt2jc8++4x27drh5ubGsWPHcHFxeePXs6+y1g6tCr76zIjO02LMmpXBVxjeoEQSpYJz587h4eHxzDBurly5UvQsm0KI5MVsNuPg4MDs2bNJnTo1H3zwgdGRXtvy5ctp27Ytjo6OrFmzhnr16hkdyapiYmKoWLEily9fZsyYMfTs2fOtLzlKsWNl4Y9iLDsuyrLjhBBCvJrJZKJfv35ERUUxceJESpRIug+TiSUqKopvvvmGcePG4eHhwZIlS8iXL5/Rsazm3r17ZM6cGScnJ8aNG0fBggUpWrRoory2XMaysvSpLasn06dK2rozf/78nDlzBrP5yUtsN27ceLw2jBBC2KOEMyI7OTk98/+cPbhw4QJeXl6MGzeOXr16ERAQkKwLnfXr11OsWDEmT54MQN26dROt0AEpdqyuget7ODm8fE0SJwdFQ9f3kyhRrPr16/Pvv/+ycOHCJ7aPHj06SXMIIURiel4jsr3ddbV27VpcXV3566+/WLZsGb/99hupUqUyOpZVhIaG0qFDB+rWrUv27Nnx9PS0ynnkMpaVdfAqyPJDV4gxm154jLOjA+29CiRhKhgwYAALFizA19eXAwcO8NFHHxEYGMju3bvJkSNHil80Tghhfx49ekTVqlUJCwtj06ZNdjdRYHR0NN999x2//PILrq6uLF26lEKFChkdy2oCAwNp2bIlly5dYsCAAXz//fekTp3aKueSYsfK8mVPz6SWbs/MswOxIzrOjg5MaulGvuzpkzRXjhw5CAwMpG/fvsycOROlFFWqVGH79u2UKVOGtGnTJmkeIYR4W6lTp2bKlCkULlzY7hqRr1y5QtOmTQkMDKRz58789ttvb3SLtT2JjIwkVapUBAQEWG1EJ54UO0mgSpFcbOzlxX8DzrEy+ArhUTGkT+VEQ9f3ae9VIMkLnXgFChRgxYoVT2y7ffs2t2/fJm/evIZkEkKI12Eymejfvz8fffQRHTt2pHbt2kZHem2bN2+mRYsWPHz4kPnz59O8eXOjI1nNgQMHOHjw4OOV2f/880+cnZ2tfl4pdpJIvuzpGd6gRJLeXv4qERERz4zgjBw5EiBZT1QlhEgeEs6I3LdvX6PjvDaTycSwYcMYPnw4xYoVY9myZXz00UdGx7KKqKgohg8fzk8//UTevHnx9fUlXbp0SVLogBQ7KVrt2rXJly8f7u7umEwmtm7dytq1a/H09KRBgwZGxxNCiBc6e/Ys9evX5/Tp00yZMsXuJgq8fv06LVq0YOvWrbRu3ZqJEyeSPr0xo/zWdvz4cVq1asWRI0do06YNY8eOJV26dEmaQYqdFKx+/frMmTOHVatWERERQZ48eejbty9DhgzB0dHR6HhCCPFcd+7coVy5cphMJrtsRN65cydNmzbl3r17zJw5E19fX6MjWc3du3fx9PQkffr0rF69ms8++8yQHFLspGB9+/a1y6FfIUTKli1bNoYPH061atXsqhHZbDYzcuRIvvvuOwoXLoy/vz8ff/yx0bGs4vr167zzzjtkzZqVuXPnUrFiRXLkyGFYHvuafEAIIUSKFN+IvGvXLgA6d+5sV4XO7du3qVevHt9++y2NGzcmKCgoWRY6ZrOZCRMmUKBAAf744w8AGjRoYGihAzKyI4QQwsaFhobSrFkz1q9fT9q0aalUqZLRkV7L3r17adKkCdevX2fSpEl07tw5Wc5ldvHiRXx9fdm2bRu1a9fG3d3d6EiPSbEjhBDCZoWEhFC/fn3++ecfu2tE1lozduxYvvnmG1xcXNizZw+lS5c2OpZVLFy4kM6dO2M2m5k+fTrt2rWzqYJOih0hhBA2KSQkBA8PD8xms901It+7d4+2bduycuVKGjRowKxZs8iSJYvRsawmKioKV1dXZs2aRYECSbsigCWkZ0cIIYRNyp8/P76+vuzfv9+uCp1Dhw7h5ubGmjVrGDNmDCtWrEiWhc6yZcvw8/MDoFWrVmzbts0mCx2QYkcIIYQNMZlMDB48mPPnz+Pg4MCoUaPsphFZa83kyZPx9PQkOjqaXbt20bt3b5u6nJMY7ty5Q/
```

Comment:
Maybe let's try plotting 20% error? That might be closer to what the systematic offset is for this particular object.